connector/src/main/java/org/geysermc/connector/network/translators/java/JavaDeclareCommandsTranslator.java

```diff
@@ -320,5 +447,14 @@ public void buildChildren(CommandNode[] allNodes) {
 
             return treeParamData;
         }
+
+        @Override
+        public String toString() {
+            return "ParamInfo{" +
+                    "paramNode=" + paramNode +
+                    ", paramData=" + paramData +
+                    ", children=" + children +
+                    '}';
+        }
```

Comment:
Won't Lombok generate this with `@ToString`?

Reply:
Yes. Yes it would.